project_projected_hours/views/project_task_progress_report.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo>       
+        <record id="view_task_project_progress_tree" model="ir.ui.view">
+            <field name="name">report.project.task.progress.tree</field>
+            <field name="model">project.task</field>
+            <field name="arch" type="xml">
+                <tree string="Tasks Progress" create="false" default_order="id desc">
+                     <field name="project_id"/>   
+                     <field name="name" string="Task"/>                  
+                     <field name="planned_hours"/>
```

Comment:
Hello @rivo2302 
Fields with hours are not displayed in the correct format. You need to add the 'timesheet_uom_no_toggle' widget and optional='show' to allow hiding of these fields when needed.